package.json

```diff
@@ -13,8 +13,11 @@
     "@tanstack/react-location-devtools": "^3.4.4",
     "@tanstack/react-location-jsurl": "^3.4.4",
     "@tanstack/react-location-simple-cache": "^3.4.4",
+    "@uiw/react-textarea-code-editor": "^2.0.1",
```

Comment:
I was wondering the same with this plugin, do you perhaps know what we are using this plugin for? Or will we use it for something?